### PR TITLE
add test for login feature

### PR DESCRIPTION
**Pull request by gurunglaxmi891**
added new scenario

### CODE REVIEW

File: tests/features/login.feature [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  Scenario: login with valid credentials
+    When user "admin" logs in with email "admin@admin.com" and password "admin123456"
+    Then user should be navigated to home page
+
+  Scenario: login with invalid credentials
+    When user "admin" logs in with email "admin@admin.com" and password "123"
+    Then error message "Invalid Credentials" should be shown
+
+  Scenario Outline: login with invalid email format
+    When the user enters email address as "<invalidemail>" and "<password>"
```
Comment (anon-pradip) [inline tests/features/login.feature]:
```suggestion
    When the user enters email address as "<email>" and "<password>"
```

File: tests/features/login.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    Then the error "<message>" should pop
+    Examples:
+        | invalidemail | password | message         |
```
Comment (anon-pradip) [inline tests/features/login.feature]:
```suggestion
        | invalid-email | password | message         |
```

File: tests/features/login.feature [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  Background:
+    Given the user "admin" has navigated to the login page
+
+  Scenario: login with valid credentials
+    When user "admin" logs in with email "admin@admin.com" and password "admin123456"
+    Then user should be navigated to home page
+
+  Scenario: login with invalid credentials
+    When user "admin" logs in with email "admin@admin.com" and password "123"
```
Comment (prashant-gurung899) [inline tests/features/login.feature]:
```suggestion
    When user "admin" tries to log in with email "admin@admin.com" and password "123"
```

**Comment by prashant-gurung899** [comment]
LGTM :+1:

**Pull request merged**